transformer_engine/pytorch/module/linear.py

```diff
@@ -338,6 +381,10 @@ def forward(
     def backward(
         ctx, grad_output: torch.Tensor
     ) -> Tuple[Union[torch.Tensor, None], ...]:
+        if isinstance(grad_output[0], Float8Tensor):
```

Comment:
Hi, @cyanguwa, this should be `grad_output` since `grad_output` is a Tensor instead of list.